tests/Rector/Contrib/CodeQuality/UnnecessaryTernaryExpressionRector/Correct/correct.php.inc

```diff
@@ -0,0 +1,40 @@
+<?php declare(strict_types=1);
+
+final class MyClass
+{
+    public function unnecessaryTernary(): bool
+    {
+        return $foo === $bar;
+    }
+
+    public function unnecessaryTernaryExpression(): bool
+    {
+        $return =  $foo === $bar;
+
+        return $return;
+    }
+
+    public function unnecessaryTernaryExpressionInverted(): bool
+    {
+        return $foo <= $bar;
+    }
+
+    public function ternaryExpressionInsideClosure(): bool
+    {
+        $result = function (): bool {
+            return $foo === 2;
+        };
+
+        return $result;
+    }
+
+    public function necessaryTernaryExpression(): bool
+    {
+        return $foo ?: $bar;
+    }
+
+    public function falsePositiveTernaryExpression(): string
+    {
+        return $foo === '' ? null : $foo;
+    }
+}
```

Comment:
This should be split to more standalone files, so it's easier to fix when it will break in the future

Reply:
How should we split?

Reply:
Oh, I see there should be kept untouched.
Let's keep it then as it is